### PR TITLE
External webview: allow parameters to be passed and received

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
This was done by updating `external-webview/index.html` to allow receiving parameters and also sending a "callback URL" to this page (the URL of the mini app).

On the external webview page, the user can also set parameters which will be returned to the JS SDK Sample App, and the sample app will display these parameters on the landing page.

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sample/src/pages/uri-schemes.js [hunk 1]
```diff
@@ -24,14 +29,41 @@ const useStyles = makeStyles((theme) => ({
...
+
+  function onOpenExternalWebview() {
+    if (params && !validateParams(params)) {
+      window.alert(
+        'Invalid params. Please input params in the format ?param1=value1&param2=value2'
+      );
+      return;
+    }
+
+    const callbackUrl = `${window.location.protocol}//${window.location.host}/index.html`;
```
Comment (minh-rakuten) [inline js-miniapp-sample/src/pages/uri-schemes.js:59]: It is more ideal to redirect to `URI Schemes` page
but this redirect to landing page is okay if we don't want to spend much effort on sample app 😄

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: js-miniapp-sample/src/pages/uri-schemes.js (hunk 1 above)
Comment (corycaywood) [reply js-miniapp-sample/src/pages/uri-schemes.js:59]: Yeah, that's true - this is much simpler to do 😆 

Also, we can re-use this for showing params that are passed to the mini app when created.

**Pull request merged**